main/webapp/modules/core/scripts/reconciliation/standard-service-panel.js

```diff
@@ -206,23 +206,60 @@ ReconStandardServicePanel.prototype._populatePanel = function() {
 
   var detailTable = $(
       '<table>' +
-      '<tr><th>'+$.i18n('core-recon/column')+'</th><th>'+$.i18n('core-recon/include')+'?</th><th>'+$.i18n('core-recon/as-property')+'</th></tr>' +
+      '<tr><th>'+$.i18n('core-recon/column')+'</th><th>'+$.i18n('core-recon/as-property')+'</th></tr>' +
       '</table>'
   ).appendTo(detailTableContainer)[0];
 
   function renderDetailColumn(column) {
     var tr = detailTable.insertRow(detailTable.rows.length);
     var td0 = tr.insertCell(0);
     var td1 = tr.insertCell(1);
-    var td2 = tr.insertCell(2);
-
-    $(td0).html(column.name);
-    $('<input type="checkbox" name="include" />')
-    .attr("columnName", column.name)
-    .appendTo(td1);
-    $('<input size="25" name="property" />')
-    .attr("columnName", column.name)
-    .appendTo(td2);
+    $(td0).attr("columnName", column.name).html(column.name);
+    $(td1).data('id','property').css('position','relative');
+
+    let mappedColumn = $("<span>").addClass("mapped-column")
+        .css("display", "none")
+        .css("width","100%")
+        .css("padding","2px")
+        .css("position","absolute")
+        .css("top","0")
+        .css("left","0")
+        .css("background",'transparent');
```

Comment:
Perhaps it would be worth migrating most (if not all) of those CSS rules to a CSS file? And similarly below.

Reply:
Yes